setup.py

```diff
@@ -26,7 +26,7 @@ def readfile(name):
 VERSION = '1.10.2'
 
 install_requires = [
-    'hupper',
+    'hupper >= 1.5',
```

Comment:
please add a comment about why it’s pinned. Something like “ignore_files support” is enough.